ntbs-integration-tests/NotificationPages/BasicEditPageTests.cs

```diff
@@ -9,38 +10,59 @@ namespace ntbs_integration_tests.NotificationPages
+        public BasicEditPageTests(NtbsWebApplicationFactory<Startup> factory) : base(factory) { }
 
-        public static IEnumerable<object[]> EditPageRoutes()
+        [Theory, MemberData(nameof(EditPageRoutes))]
+        public async Task Get_ReturnsNotFound_ForNewId(string route)
         {
-            yield return new object[] { Routes.Patient, Utilities.DRAFT_ID, HttpStatusCode.OK };
-            yield return new object[] { Routes.Patient, Utilities.NEW_ID, HttpStatusCode.NotFound };
-            yield return new object[] { Routes.Episode, Utilities.DRAFT_ID, HttpStatusCode.OK };
-            yield return new object[] { Routes.Episode, Utilities.NEW_ID, HttpStatusCode.NotFound };
-            yield return new object[] { Routes.ClinicalDetails, Utilities.DRAFT_ID, HttpStatusCode.OK };
-            yield return new object[] { Routes.ClinicalDetails, Utilities.NEW_ID, HttpStatusCode.NotFound };
-            yield return new object[] { Routes.ContactTracing, Utilities.DRAFT_ID, HttpStatusCode.OK };
-            yield return new object[] { Routes.ContactTracing, Utilities.NEW_ID, HttpStatusCode.NotFound };
-            yield return new object[] { Routes.SocialRiskFactors, Utilities.DRAFT_ID, HttpStatusCode.OK };
-            yield return new object[] { Routes.SocialRiskFactors, Utilities.NEW_ID, HttpStatusCode.NotFound };
-            yield return new object[] { Routes.Travel, Utilities.DRAFT_ID, HttpStatusCode.OK };
-            yield return new object[] { Routes.Travel, Utilities.NEW_ID, HttpStatusCode.NotFound };
-            yield return new object[] { Routes.Comorbidities, Utilities.DRAFT_ID, HttpStatusCode.OK };
-            yield return new object[] { Routes.Comorbidities, Utilities.NEW_ID, HttpStatusCode.NotFound };
-            yield return new object[] { Routes.Immunosuppression, Utilities.DRAFT_ID, HttpStatusCode.OK };
-            yield return new object[] { Routes.Immunosuppression, Utilities.NEW_ID, HttpStatusCode.NotFound };
-            yield return new object[] { Routes.PreviousHistory, Utilities.DRAFT_ID, HttpStatusCode.OK };
-            yield return new object[] { Routes.PreviousHistory, Utilities.NEW_ID, HttpStatusCode.NotFound };
+            // Act
+            var response = await client.GetAsync($"{route}?id={Utilities.NEW_ID}");
+
+            // Assert
+            Assert.Equal(HttpStatusCode.NotFound, response.StatusCode);
         }
 
-        [Theory, MemberData(nameof(EditPageRoutes))]
-        public async Task Get_ReturnsOkOrNotFound_DependentOnId(string route, int id, HttpStatusCode code)
+        [Theory, MemberData(nameof(OkRouteToIdCombinations))]
+        public async Task Get_ReturnsOk_ForExistingIds(string route, int id)
         {
-            // Act
-            var response = await client.GetAsync($"{route}?id={id}");
+            //Act
+            var response = await client.GetAsync($"{route[0]}?id={id}");

```

Comment:
why isn't this just "route"? Wouldn't this just be the first character of route?

Reply:
Very good catch, pre-push test didn't fail this as first character of route is a slash.

As to why it's not just route, it's a remnant of a previous refactor with a nested foreach adding a base object[] to another object[]...